### DISCUSSION TITLE
PCL2启动器出现问题

### DISCUSSION THREAD

**Discussion by Berver2010** [Meloong-Git/PCL#1534] [提问] [answered]
启动游戏后360安全卫士弹出警告，显示“这是敲诈病毒，回勒索你的钱财”
不管是拒绝执行还是允许执行，都会游戏和启动器都一起崩溃

**Comment by Berver2010** [comment]
是2.5.2的版本

**Comment by WForst-Breeze** [comment]
您是否将 PCL2 及相关进程添加至白名单？若没有，请将其添加至白名单后再尝试启动并查看问题是否仍然存在

**Comment by LTCatt** [comment, marked as answer]
360 请自行添加白名单使用（
如果崩溃了那也特么是 360 导致的……
